include/llvm/IR/Instruction.h

```diff
@@ -518,12 +529,7 @@ class Instruction : public User, public ilist_node<Instruction> {
 // HLSL Change Starts
 // Temporarily disable "downcast of address" UBSAN runtime error
 // https://github.com/microsoft/DirectXShaderCompiler/issues/6446
-#ifdef __has_feature
-#if __has_feature(undefined_behavior_sanitizer)
-__attribute__((no_sanitize("undefined")))
-#endif // __has_feature(address_sanitizer)
-#endif // defined(__has_feature)
-// HLSL Change Ends
+#if 0
```

Comment:
Alternatively we can delete this whole overload.  It's done inline above at line 43, and differently.
Not sure what you wanted.

Reply:
I'd be fine with just deleting the code.

Reply:
I plan to delete the code on a followup when addressing the other cases.